docs/source/api.rst

```diff
@@ -192,12 +191,12 @@ Atom entry, so the following Atom wrapper XML tree is created::
     <entry xmlns="http://www.w3.org/2005/Atom">
         <title>PREMIS event entry for object_123</title>
         <id>PREMIS event entry for object_123</id>
```

Comment:
`title` and `id` values here could be changed to go along with the changes made to the premis event example above.